src/components/core/Flex/Flex.js

```diff
@@ -1,11 +1,9 @@
 import styled from 'styled-components';
 import Box from '../Box';
 
-export const Flex = styled(Box)({});
-
-Flex.defaultProps = {
+export const Flex = styled(Box)({
```

Comment:
This will break `display="inline-flex"`.

Reply:
Hmm, why will it?

Reply:
How do you set `inline-flex`? Either ways sounds like inline could be a separate component.

Reply:
I feel it's used too rarely to have its own component, and styled-system has the `display` prop, but it can't override component styles.

Reply:
Sgtm for now but see my reasoning below. It applies here as well.